index.html

```diff
@@ -4,16 +4,26 @@
     <meta charset="UTF-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <meta http-equiv="X-UA-Compatible" content="ie=edge"/>
-    <title>Title</title>
-    <link rel="stylesheet" href="styles/App.scss"/>
+    <title>Movie</title>
+    <link rel="stylesheet" href="styles/App.css"/>
 
     <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
           rel="stylesheet">
 </head>
 <body>
+    <div id="header">
+        <h3 id="web_title">TITLE</h3>
```

Comment:
Generally the standard is to use ID for selecting an element from DOM using JS, and for styling we should be using the class

Here I think is kinda minor, because we will only have one footer and header on the page to it can be unique

but this <h3 /> element shouldn't be